chipsec_main.py

```diff
@@ -399,6 +399,9 @@ def parse_args(self):
             logger().DEBUG   = True
         if self.log:
             logger().set_log_file( self.log )
+        if self._module_argv and len(self._module_argv) == 1 and len(self._module_argv[0].split(',')) > 1:
```

Comment:
You should be using `self._module_argv[0].count(',') >= 1` instead of splitting and generating a new list.